tsuru_dashboard/static/css/abyss.login.css

```diff
@@ -43,15 +70,59 @@ body {
             box-sizing: border-box;
 }
 .form-signin div.input-wrapper > ul.errorlist {
-	background: #FFC4C4;
-	border: 1px solid #C20000;
-	margin: 0;
+	margin-top: 2px;
+  color: #d40b0b;
+	padding: 0;
+  position: relative;
+  top: 66px;
 }
 .form-signin div.input-wrapper > ul.errorlist li {
 	list-style: none;
-	padding: 5px;
 	font-size: 0.9em;
 }
 .form-actions {
 	margin: 0px;
 }
+.form-actions .button-login {
+	text-transform: uppercase;
+  color: #fff;
```

Comment:
There are still a couple of lines with wrong indentation. Can you check this for the whole file?